### PR TITLE
ci: install emulator dependencies using requirements file

### PR DESCRIPTION
**Pull request by coryan**
Avoids duplicating the dependencies in Dockerfiles and the `requirements.txt` files.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/6457)

### CODE REVIEW

File: ci/cloudbuild/builds/lib/integration.sh [hunk 1]
```diff
@@ -196,3 +196,7 @@ function integration::ctest_with_emulators() {
     "./google/cloud/bigtable/ci/${EMULATOR_SCRIPT}" \
     "${cmake_out}" "${ctest_args[@]}" -L integration-test-emulator
 }
+
+# To run the integration tests we need to install the dependencies for the storage emulator
+export PATH="${HOME}/.local/bin:${PATH}"
```
Comment (devjgm) [inline ci/cloudbuild/builds/lib/integration.sh:201]: I'd move this up top to just after the `source module` lines to keep all the code that executes automatically in this file in one spot.

**Review by devjgm — APPROVED** [review]

File: ci/cloudbuild/builds/lib/integration.sh (hunk 1 above)
Comment (coryan) [reply ci/cloudbuild/builds/lib/integration.sh]: Done.

**Review by devjgm — APPROVED** [review]

**Comment by frankyn** [comment]
Thanks @coryan!

**Pull request merged**